ui/widgets/autocomplete.js

```diff
@@ -223,6 +223,9 @@ $.widget( "ui.autocomplete", {
 			.menu( "instance" );
 
 		this._addClass( this.menu.element, "ui-autocomplete", "ui-front" );
+		if ( this.isRtl() ) {
+			this._addClass( this.menu.element, "ui-autocomplete-menu-rtl" );
```

Comment:
agreed on generic but i wonder should we even use a class or should we set the attribute?